internal/api/commitment.go

```diff
@@ -762,3 +763,77 @@ func (p *v1Provider) TransferCommitment(w http.ResponseWriter, r *http.Request)
 
 	respondwith.JSON(w, http.StatusAccepted, map[string]any{"commitment": c})
 }
+
+// GetCommitmentConversion handles GET /v1/commitment-conversion/{service_type}/{resource_name}
+func (p *v1Provider) GetCommitmentConversions(w http.ResponseWriter, r *http.Request) {
+	httpapi.IdentifyEndpoint(r, "/v1/commitment-conversion/:service_type/:resource_name")
+	token := p.CheckToken(r)
+	if !token.Require(w, "cluster:show_basic") {
+		return
+	}
+
+	sourceBehavior := p.parseAndValidateConversionRequest(w, r)
+	if sourceBehavior == nil {
+		return
+	}
+	sourceAPIIdentity := sourceBehavior.IdentityInV1API
+
+	conversions := make([]limesresources.CommitmentConversion, 0)
+	for targetService, quotaPlugin := range p.Cluster.QuotaPlugins {
+		sourceResource := quotaPlugin.Resources()[sourceAPIIdentity.ResourceName]
+		for targetResourceName := range quotaPlugin.Resources() {
+			targetResource := quotaPlugin.Resources()[targetResourceName]
+			targetBehavior := p.Cluster.BehaviorForResource(targetService, targetResourceName)
+			targetAPIIdentity := targetBehavior.IdentityInV1API
+			if targetBehavior.CommitmentConversion == (core.CommitmentConversion{}) {
+				continue
+			}
+			if sourceAPIIdentity.ServiceType == targetAPIIdentity.ServiceType && sourceAPIIdentity.ResourceName == targetAPIIdentity.ResourceName {
+				continue
+			}
+			if sourceResource.Unit != targetResource.Unit {
+				continue
+			}
+			if sourceBehavior.CommitmentConversion.Identifier != targetBehavior.CommitmentConversion.Identifier {
+				continue
+			}
+
+			from, to := p.getCommitmentConversionRate(sourceBehavior, &targetBehavior)
+			targetResourceName := limesresources.ResourceName(string(targetAPIIdentity.ServiceType) + "/" + string(targetAPIIdentity.ResourceName))
+			result := limesresources.CommitmentConversion{
+				From:           from,
+				To:             to,
+				TargetResource: targetResourceName,
+			}
+			conversions = append(conversions, result)
+		}
+	}
+
+	respondwith.JSON(w, http.StatusOK, map[string]any{"conversions": conversions})
+}
+
+func (p *v1Provider) parseAndValidateConversionRequest(w http.ResponseWriter, r *http.Request) *core.ResourceBehavior {
+	serviceType := limes.ServiceType(mux.Vars(r)["service_type"])
+	resourceName := liquid.ResourceName(mux.Vars(r)["resource_name"])
```

Comment:
Although I just said that I'm going to fix this more broadly in the next refactoring, this should use the API-level type just to avoid future confusion:
```suggestion
	resourceName := limesresources.ResourceName(mux.Vars(r)["resource_name"])
```